F7 broken in latest development

gcc-arm-none-eabi-6-2017q2
`make TARGET=OMNIBUSF7`

```
%% pwm_output.c
./src/main/drivers/pwm_output.c: In function 'pwmOutConfigTimer':
./src/main/drivers/pwm_output.c:122:31: warning: 'return' with a value, in function returning void
     if (Handle == NULL) return p;
                               ^
./src/main/drivers/pwm_output.c:118:13: note: declared here
 static void pwmOutConfigTimer(pwmOutputPort_t * p, const timerHardware_t *timerHardware, uint8_t mhz, uint16_t period, uint16_t value)
             ^~~~~~~~~~~~~~~~~
```
 (arm tools 4.8 throws an error in the same case build is terminated)

Result:

boards seems to be working correctly with gcc6

Looks like it was introduced by https://github.com/iNavFlight/inav/commit/a53b6d78b38fd484e276369ef565a28c6c38c08c

--- a/src/main/drivers/pwm_output.c
+++ b/src/main/drivers/pwm_output.c
@@ -119,7 +119,7 @@ static void pwmOutConfigTimer(pwmOutputPort_t * p, const timerHardware_t *timerH
 {
 #if defined(USE_HAL_DRIVER)
     TIM_HandleTypeDef* Handle = timerFindTimerHandle(timerHardware->tim);
-    if (Handle == NULL) return p;
+    if (Handle == NULL) return;
 #endif
 
     configTimeBase(timerHardware->tim, period, mhz);

